File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
import java.util.stream.IntStream;
import java.util.Arrays;

class Solution {
    public int[] solution(int[] numbers, String direction) {
        int[] frstArr;
        int[] nxtArr;
        
        if ("right".equals(direction)) {
            frstArr = new int[]{numbers[numbers.length - 1]};
            nxtArr = Arrays.copyOfRange(numbers, 0, numbers.length - 1);
        }
        else {
            frstArr = Arrays.copyOfRange(numbers, 1, numbers.length);
            nxtArr = new int[]{numbers[0]};
        }
        
        return IntStream.concat(
            Arrays.stream(frstArr), Arrays.stream(nxtArr)
        )
        .toArray();
    }
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.js
function solution(numbers, direction) {
    if ('right' === direction) {
        numbers.splice(0, 0, numbers[numbers.length - 1]);
        return numbers.slice(0, numbers.length - 1);
    }
    else {
        numbers.splice(numbers.length, 0, numbers[0])
        return numbers.slice(1, numbers.length);
    }
}